editor-core/src/main/webapp/app/core/editorCore/model/DefaultDiagramNode.ts

```diff
@@ -99,17 +102,35 @@ export class DefaultDiagramNode implements DiagramNode {
         }
     }
 
+    setPropertyEditElementsPosition(x : number, y : number) : void {
+        let propertiesCount = 0;
+
+        for (let i in this.propertyEditElements) {
+            this.propertyEditElements[i].setPosition(x, y - this.delta*propertiesCount);
+            propertiesCount++;
+        }
+    }
+
     initPropertyEditElements(zoom: number): void {
         var parentPosition = this.getJointObjectPagePosition(zoom);
-        this.propertyEditElement = new PropertyEditElement(this.logicalId, this.jointObject.id,
-            this.changeableProperties);
         var propertyEditElementX = parentPosition.x + (<number> (this.boundingBox.width - 50)/2);
-        var propertyEditElementY = parentPosition.y + this.boundingBox.height - 50;
-        this.propertyEditElement.setPosition(propertyEditElementX, propertyEditElementY);
+        var propertyEditElementY = parentPosition.y + this.boundingBox.height - this.topIndent;
+
+        for (var propertyKey in this.changeableProperties) {
```

Comment:
As IDEA reminds: it's better to use `let` instead of `var` cause it behaves more strictly.